include/deal.II/lac/trilinos_solver.h

```diff
@@ -114,25 +114,19 @@ namespace TrilinosWrappers
        * base class for all solvers.
        */
       explicit
-      AdditionalData (const bool         output_solver_details   = false,
-                      const unsigned int gmres_restart_parameter = 30);
```

Comment:
And then you would again get the problem, that SolverBase doesn't use that object in the do_solve routine where it needs the options. At least i have tried it and couldn't find a way getting that to work. I also get two structs of AdditionalData (one from the inherited and one from SolverBase) in the SolverCG when i inspect the object in a debugger. Maybe i don't understand something here.